flows/events/session_triggered.go

```diff
@@ -40,6 +40,11 @@ const TypeSessionTriggered string = "session_triggered"
 //           "created_on": "2000-01-01T00:00:00.000000000-00:00"
 //         }
 //       }
+//     },
+//     "history": {
+//       "parent_uuid": "55105da5-abb5-4690-b1f6-ec2e5762a561",
+//       "ancestors": 3,
+//       "ancestors_since_input": 1
```

Comment:
@nicpottier so I don't think just having this on the event is going to be enough - for the IVR case this would need to be saved to `flows_flowstart`.. which could be achieved a few different ways..

 1. Each of these things becomes a field on `flows_flowstart` (great for querying, maybe useful..)
 2. Add new JSON column to hold the history blob
 3. Save the entire `session_triggered` event to a new blob field - eventually it would replace use of `parent_summary`

Reply:
I *think* I prefer 2, but I don't feel super strongly on these.